src/Microsoft.Extensions.Logging.Abstractions/NullLoggerFactory.cs

```diff
@@ -3,15 +3,20 @@
 
 namespace Microsoft.Extensions.Logging.Abstractions
 {
+    /// <summary>
+    /// An <see cref="ILoggerFactory"/> used to create instance of <see cref="NullLogger"/>.
```

Comment:
Might be worth adding some text that explicitly states that this logger never logs anything? Same goes for the methods below...